src/main/java/me/skulduggerry/gameoflife/controller/GenerationController.java

```diff
@@ -0,0 +1,49 @@
+package me.skulduggerry.gameoflife.controller;
+
+import java.io.IOException;
+
+import javax.servlet.http.HttpServletResponse;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestPart;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+import lombok.extern.slf4j.Slf4j;
+import me.skulduggerry.gameoflife.model.Generation;
+
+@RestController
+@Slf4j
+public class GenerationController {
+
+    @Autowired
+    private ObjectMapper mapper;
+
+    @PostMapping("/upload")
+    public String upload(@RequestPart("file") MultipartFile file, HttpServletResponse response) throws IOException {
```

Comment:
The exception should be mapped into a response aswell. Have a look at the way we're doing it in other services, e.g.
https://github.com/gbtec-ag/biccloud-method-service/blob/development/src/main/java/com/gbtec/bic/cloud/method/rest/ExceptionHandler.java

We probably don't need all of the response statuses from that example - choose the appropriate ones.